CIP-high-assurance/README.md

```diff
@@ -0,0 +1,173 @@
+---
+CIP: XXX
+Title: High Assurance Best Practices
+Category: Meta
+Status: Draft
```

Comment:
```suggestion
Status: Proposed
```
There is no `Draft` status & at the time of merged this will be `Proposed` if not already `Active` at that time.